python/test/unit/hopper/test_experimental_tma.py

```diff
@@ -1,56 +1,24 @@
 import pytest
 import torch
-import tempfile
 
 import triton
 import triton.language as tl
 from triton.tools.experimental_descriptor import create_1d_tma_descriptor, create_2d_tma_descriptor
 
 
-def test_descriptor_load_ttgir():
```

Comment:
Note: @ThomasRaoux advised that this test should be deleted, as it was added back when the op was only exposed at TTGIR level.